<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Renan Sauteraud</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">

    <!-- Le styles -->
    <link href="./bootstrap/css/bootstrap.css" rel="stylesheet">
    <link href="./bootstrap/css/bootstrap.min.css" rel="stylesheet">
    <link href="./font-awesome/css/font-awesome.min.css" rel="stylesheet">
    <!-- my css <link href="./utils/css/rglab.css" rel="stylesheet">-->
	
    <!-- Le fav and touch icons -->
    <!-- <link rel="shortcut icon" href="images/favicon.ico">
    <link rel="apple-touch-icon" href="images/apple-touch-icon.png">
    <link rel="apple-touch-icon" sizes="72x72" href="images/apple-touch-icon-72x72.png">
    <link rel="apple-touch-icon" sizes="114x114" href="images/apple-touch-icon-114x114.png"> -->
  </head>
  
  <!--R All displayed content should be in body-->
  <body style="padding-top: 50px;"><!--50 so the body starts under the navbari-fixed-top-->
    <div class="navbar navbar-inverse navbar-fixed-top">
      <div class="navbar-inner">
        <div class="container">
          <a class="brand" href="./index.html">Renan Sauteraud</a>
        </div>
      </div>
    </div><!--navbar-->


    <div class="container">
      <div class="row">
        <div class="span12">
          <div class="well">
            <p>
              I currently work as a scientific programmer at the <a href="http://www.fhcrc.org">Fred Hutchinson Cancer Research Center</a> in the <a href="http://www.rglab.org">RGLab</a> since April 2012. I am mostly developing and maintaining R packages but am also experienced with C/C++, python and Java.
            <br>
            <br>
              In 2011, I completed my master degree in <a href="http://www.bibs.u-psud.fr/master.php" title="Careful! It's in French">Bioinformatics and Biostatistics</a> at <a href="http://www.u-psud.fr/fr/index.html">Universit&eacute; Paris-Sud</a> xxcfdgffsl;gsdkg;lsafter a master thesis in the <a href="http://www.dbb.su.se/">Department of Biochemistry and Biophysics</a> at Stockholms universitet under the supervision of <a href="http://www.dbb.su.se/en/?p=researchgroup&id=37">Prof. Arne Elofsson</a>. 
            </p>
          </div>
        </div>
      </div>
    </div><!--container-->

    <div class="container" id="Rpackages">
      <div class="row">
        <div class="span12">
          <h2>Software</h2>
          <p>Following is a list of <a href="http://www.r-project.org/">R</a> packages I am currently maintaining. These packages are available on <a href="https://github.com/SRenan">my GitHub page</a> or the <a href="https://github.com/RGLab">RGLab's GitHub page</a>.
          <br><br></p>
        </div>
      </div>
      <div class="row">
        <div class="span8">
          <h4>PICS</h4>
          <p>For the analysis of ChIP-Seq data, PICS (Probabilistic Inference for ChIP-Seq) identifies genomic regions bound by transcription factors.
          The algorithm is implemented in C and makes use of parallel computing.</p>
          <p><a href="http://www.bioconductor.org/packages/release/bioc/html/PICS.html">Bioconductor</a>, <a href="https://github.com/SRenan/PICS">GitHub</a><p>
          <h4>PING</h4>
          <p><a href="http://www.bioconductor.org/packages/release/bioc/html/PING.html">Bioconductor</a>, <a href="https://github.com/SRenan/PING">GitHub</a><p>
          <h4>pepStat</h4>
          <p>This package predicts antibody binding sites on peptides using microarray data.</p>
          <p><a href="https://github.com/RGLab/pepStat">GitHub</a><p>
          <h4>PEP.db</h4>
          <p><a href="https://github.com/RGLab/PEP.db">GitHub</a><p>
          <h4>Pviz</h4>
          <!--<p>Pviz is a package for peptide visualisation inspired from the popular <a href="http://www.bioconductor.org/packages/release/bioc/html/Gviz.html">Gviz</a>. It add new tracks for visualisation of peptide microarray data and protein annotations while staying fully compatible with Gviz.</p>-->
          <p>Pviz is an R package for peptide visualisation inspired by and depending on <a href="http://www.bioconductor.org/packages/release/bioc/html/Gviz.html">Gviz</a>. It introduces new types of track and extends the existing ones in order to deal with amino-acid based data such as peptide microarray.
          <p><a href="https://github.com/RGLab/Pviz">GitHub</a><p>
          <h4>HIV.db</h4>
          <p>HIV.db is a collection of annotations for HIV (hxb2) and SIV (mac239) sequences. It includes genes, gene products, genomic structure elements as well as known epitopes. The selected elements can be plotted against the data using Pviz</p>
          <p><a href="https://github.com/RGLab/HIV.db">GitHub</a><p>
          <h4>LumiR</h4>
          <p><a href="https://github.com/RGLab/LumiR">GitHub</a><p>
        </div>
        <div class="span4 text-center">
          <img src="./images/PING.png" alt="PING plot" class="img-rounded" style="border: 1px solid #e0e0e0">
          <p style="font-size: 75%">Summary plot of a <b>PING</b> analysis</p>
          <br>
          <img src="./images/LumiR.png" alt="LumiR plot" class="img-rounded" style="border: 1px solid #e0e0e0">
          <p style="font-size: 75%">Fitting of 5-Parameter Logistic curves using <b>LumiR</b></p>
        </div>
      </div>
    </div><!--Rpackages"-->

    <div class="container" id="publications">
      <div class="row">
        <div class="span12">
          <h2>Publications</h2>
          <ul>
          <li><p>Greg C. Imholte, Renan Sauteraud, Bette Korber, Robert T. Bailer, Ellen T. Turk, Xiaoying Shen, Georgia D. Tomaras, John R. Mascola, Richard A. Koup, David C. Montefiori, Raphael Gottardo
            <b>A computational framework for the analysis of peptide microarray antibody binding data with application to HIV vaccine profiling</b>
            <em>Journal of Immunological Methods</em>, Volume 395, Issues 1–2, 30 September 2013, Pages 1–13
            <a href="http://dx.doi.org/10.1016/j.jim.2013.06.001"class="icon-file" target="_BLANK"></a>
          </p></li>
          <li><p>Sangsoon Woo, Xuekui Zhang, Renan Sauteraud, Francois Robert, Raphael Gottardo.
            <b>PING 2.0: an R/Bioconductor package for nucleosome positioning using next-generation sequencing data</b>.
            <em>Bioinformatics</em> 29(16): 2049-2050 (2013)
            <a href="http://bioinformatics.oxfordjournals.org/content/29/16/2049" class="icon-file" target="_BLANK"></a>
          </p></li>
          </ul>
          <hr>
        </div>
      </div>
    </div><!--publications-->

    <div class="container" id="contact">
      <div class="row">
        <div class="span12">
          <h2>Contact</h2>
          <ul>
          <li><i class="icon-github"></i><b> GitHub: </b><a href="http://www.github.com/SRenan">github.com/SRenan</a></li>
          <li><i class="icon-linkedin-sign"></i><b> LinkedIn: </b><a href="http://www.linkedin.com/pub/renan-sauteraud/47/91b/6b0">linkedin.com/renan-sauteraud</a></li>
          <li><i class="icon-twitter"></i><b> Twitter: </b><a href="https://twitter.com/Srenan_">@SRenan_</a></li>
          </ul>
          <hr>
        </div>
      </div>
    </div><!--contact-->

    <!-- == Footer == -->
    <div class="container">
      <div class="row">
        <div class="span12">
          <footer class="footer">
            <p class="pull-right"><a href="#">Back to top</a></p>
            <p class="pull-left">Made with <a href="http://twitter.github.com/bootstrap">Bootstrap</a>.</p>
          </footer>
        </div>
      </div>
    </div><!--container-->

    <!-- Le javascript -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script type="text/javascript" src="http://platform.twitter.com/widgets.js"></script>	
    <script src="./bootstrap/js/bootstrap.js"></script>
  </body>
</html>
